SPECS/ocaml-base/ocaml-base.spec

```diff
@@ -1,18 +1,16 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
 # TESTING NOTE: The ppx_jane module is needed to run the tests.  However,
 # ppx_jane transitively requires this module.  Therefore, we cannot run the
 # tests at all until we are able to add ppx_jane to Fedora, and even then we
```

Comment:
Nit: Fedora-specific comment here that could be removed

Reply:
I really appreciate catching lingering Fedora-specific comments. I managed to catch most of the macros, which we don't use in Mariner, but comments were a bit more tricky. I'll update them soon.